### ISSUE TITLE
[0.8] host, port, etc are reserved for anchor elements; let's avoid them

### ISSUE THREAD

**Issue by nevir** [Polymer/polymer#1417]
Ends up causing really funky things to occur if you extend `a` (such as overwriting the host portion of the linked URL)

```js
Polymer({
  is: 'super-link',
  extends: 'a',
  ready() {
    this.setAttribute('href', '#awesome');
    // When we set `this.host` up the stack, we screw up the href
  },
});
```

**Comment by Punk-UnDeaD** [comment]
in Firefox all ok
but Chrome failed

**Comment by sorvell** [comment]
Fixed in version 0.9
